File: Multi_Language_CodeExamples/php/calculator.php
<?php
class Calculator {
    public function add($a, $b) {
        return $a + $b;
    }
    
    public function multiply($a, $b) {
        return $a * $b;
    }
}

$calc = new Calculator();
echo "5 + 3 = " . $calc->add(5, 3) . "\n";
echo "5 * 3 = " . $calc->multiply(5, 3) . "\n";
?>

File: Multi_Language_CodeExamples/java/Calculator.java
public class Calculator {
    public static int add(int a, int b) {
        return a + b;
    }
    
    public static int multiply(int a, int b) {
        return a * b;
    }
    
    public static void main(String[] args) {
        System.out.println("5 + 3 = " + add(5, 3));
        System.out.println("5 * 3 = " + multiply(5, 3));
    }
}
